installer/tue-install-impl.bash

```diff
@@ -36,7 +36,7 @@ Error while installing target '$TUE_INSTALL_CURRENT_TARGET':
 
     $1
 \033[0m"
-    return 1
+    exit 1
```

Comment:
Do you really want to break out of the installation if one target fails?

Reply:
It is best thing to do in my opinion. Because it makes it very clear where the error occurs.